community/tools/ra-data-hasura/src/index.js

```diff
@@ -42,8 +66,11 @@ export default (serverEndpoint, headers) => {
         finalSelectQuery.args.limit = params.pagination.perPage;
         finalSelectQuery.args.offset = (params.pagination.page * params.pagination.perPage) - params.pagination.perPage;
         finalSelectQuery.args.where = params.filter;
-        finalSelectQuery.args.order_by = {column: params.sort.field, type: params.sort.order.toLowerCase()};
+        // finalSelectQuery.args.order_by = {column: params.sort.field, type: params.sort.order.toLowerCase()};
```

Comment:
Any reason for leaving the old commented code in? This is generally considered a bad practice. If someone wants to refer to the previous version, they can look up the history using `git`.

References:

- https://stackoverflow.com/questions/3664288/is-it-bad-practice-to-leave-commented-out-code-in-production-releases
- https://www.nayuki.io/page/dont-share-commented-out-code
- https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Reply:
Removed old comments. The PR was open for testing for @mnlbox to try it out :) So hadn't removed the old code temporarily.